### PR TITLE
Added Super Smash Bros. API

### PR DESCRIPTION
**Pull request by ASC95**
Thank you for taking the time to work on a Pull Request for this project!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] Your additions are ordered alphabetically
- [x] Your submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column should be padded with one space on either side
- [x] You have searched the repository for any relevant issues or pull requests
- [x] Any category you are creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -346,6 +346,7 @@ API | Description | Auth | HTTPS | CORS | Link |
 | Qriusity | Quiz/Trivia Questions | No | Yes | Unknown | [Go!](https://qriusity.com/) |
 | Rick and Morty | All the Rick and Morty information, including images | No | Yes | Yes | [Go!](https://rickandmortyapi.com) |
 | Riot Games | League of Legends Game Information | `apiKey` | Yes | Unknown | [Go!](https://developer.riotgames.com/) |
+| Smashdatabase | Super Smash Bros. gifs, videos, and images | `apiKey` | No | Unknown | [Go!](http://api.smashdatabase.com/) |
```
Comment (davemachado) [inline README.md]: Where does one get an API key?

Comment (ASC95) [reply README.md]: Email qualzz20@gmail.com

**Comment by ASC95** [comment]
Get an API key by emailing qualzz20@gmail.com with project information.

**Comment by davemachado** [comment]
Is this documented anywhere?

**Comment by ASC95** [comment]
It is documented at http://api.smashdatabase.com/ at the bottom of the web page.

**Comment by davemachado** [comment]
Unfortunately, this does not appear to be a self-sufficient service at this time. Emailing a single developer's email address to manually receive a key does not bode well for the "public" nature of this collection of services. I'd encourage you to open more PRs in the future!

**Pull request closed without merging**